### ISSUE TITLE
Bug: Election results dark mode text accessibility 

### ISSUE THREAD

**Issue by 2nPlusOne** [rit-sse/WebsiteTheSSEquel#532]
## Summary

The text on the election results page is black in dark mode, resulting in poor contrast with the dark backgorund.

## Impact

- Severity: Low
- Affected users/surfaces: Election results page

## Steps to Reproduce

1. View election results flow in dark mode

## Environment

- Environment: [local/dev/staging/prod]
- Browser/OS:
- Commit/branch:

## Evidence

- Screenshots/video:

<img width="879" height="581" alt="Image" src="https://github.com/user-attachments/assets/d87676d1-7f3b-40ea-8354-d25c94780278" />

<img width="1034" height="472" alt="Image" src="https://github.com/user-attachments/assets/8aff15bc-9db4-45fc-bd1b-f622c717b02a" />

## Proposed Fix (Optional)

Make the text light.
